com.gluon.scenebuilder.metainfo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.gluon.scenebuilder</id>
```

Comment:
`gluon.com` doesn't seem to be related. Is this app-id meeting the requirements?

Reply:
should replace by github.io ?

Reply:
there seems to be a `gluonhq.com` from which you download screenshot in the appdata file. That would be appropriate.